src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1523,8 +1523,9 @@ public static void EmptyJsonIsInvalid()
         }
 
         [Fact]
-        public static void EmptyJsonWithinSequenceIsInvalid(ReadOnlySpan<byte> dataUtf8)
+        public static void EmptyJsonWithinSequenceIsInvalid()
         {
+            var dataUtf8 = ReadOnlySpan<byte>.Empty;
```

Comment:
I think this test needs to use `ReadOnlySequence<byte> sequence = JsonTestHelper.GetSequence(new byte[0], 1);`